content/en/docs/methods/kfctl/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "kfctl"
+description = "Documentation of how to use the kfctl method for Kubeflow"
```

Comment:
hey @RFMVasconcelos, since it's all docs here under `/docs`, maybe there's no need for saying "Documentation":

```suggestion
description = "How to use the kfctl method for Kubeflow"
```